CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+
+## 1.4.9
+
+## Improvements
+- Native Android agent updated to version 7.6.8
+- Native iOS agent updated to version 7.5.8
+- fixed Unity Exception stack traces issue when namespaces are used.
+
+## 1.4.8
+
+## Improvements
+- Native Android agent updated to version 7.6.7
+- Native iOS agent updated to version 7.5.6
```

Comment:
iOS was updated to 7.5.8

Reply:
this is from the last release. this is 1.4.9

Reply:
ok approved.